content/_index.md

```diff
@@ -25,14 +25,11 @@ Microsoft, Amazon, and Meta.
 
 ## News
 
-* 2024-10-25: "FOSDEM 2025" - [CFP for the Image-Based Linux and Boot Integrity Devroom is now open](https://uapi-group.org/docs/conferences/2025-02-02__fosdem-devroom/)
-
-* 2024-09-26: "All Systems Go! 2024" - Video recordings released. <br />
+* 2025-10-01: "All Systems Go! 2024" — Video recordings released. <br />
   All Systems Go, the foundational user-space Linux conference, has published recordings of all talks.
-  Head over to https://www.youtube.com/playlist?list=PLWYdJViL9EipIImmvuoGFAeS-lKeHH2DD to check them out!
+  Head over to https://www.youtube.com/playlist?list=PLWYdJViL9EiqX25Rmu49FTDDnzT30wvqL to check them out!
```

Comment:
Can we use C3VOC for these links? Avoid having people visit youtube for the videos?

Reply:
We certainly can. UPdated.

Reply:
https://github.com/uapi-group/docs/pull/40 also.